cmd/root.go

```diff
@@ -22,13 +22,32 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/utils"
 )
 
+// The default log level
+var logLevel = 1
+
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "ec",
 	Short: "Tool to enforce enterprise contracts",
 	Long:  `TODO: description`,
+
+	// Prevent usage instructions showing on every error
+	SilenceUsage: true,
```

Comment:
:+1: I thought it was a good incentive (seeing the usage being printed for every error) to do error handling ([HACBS-633](https://issues.redhat.com/browse/HACBS-633)), but we can do this as well even though error handling might not result in errors reaching cobra

Reply:
It's done in #67 I noticed.